### ISSUE TITLE
Wrong sunrise and sunset

### ISSUE THREAD

**Issue by cristibaluta** [haraldh/SunCalc#4]
Hi, I compiled this but it gives totally wrong values, like 12:39PM sunrise and 6PM sunset, which is not even possible on any region i believe. Sunset is almost correct but sunrise is 5h too late. Does it work for you?

**Comment by haraldh** [comment]
Oh, it might be in the polar regions? Did you get a good GPS signal?

**Comment by cristibaluta** [comment]
Yes, it is in more remote places, you can try Iceland for example
lat = 64.371607;
lon = -20.975488;
